src/main/java/com/tenx/ms/retail/product/domain/ProductEntity.java

```diff
@@ -0,0 +1,53 @@
+package com.tenx.ms.retail.product.domain;
+import com.tenx.ms.retail.order.domain.OrderProductEntity;
+import com.tenx.ms.retail.stock.domain.StockEntity;
+import com.tenx.ms.retail.store.domain.StoreEntity;
+import lombok.Data;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+import java.math.BigDecimal;
+import java.util.List;
+
+@Entity
+@Table(name = "product")
+@Data
+public class ProductEntity {
+    @Id
+    @GeneratedValue
+    @Column(name = "product_id", nullable = false)
+    private Long productId;
+
+    @Column(name = "name", nullable = false)
+    private String name;
+
+    @Column(name = "description", nullable = false)
+    private String description;
+
+    @Column(name = "sku", nullable = false)
+    private String sku;
+
+    @Column(name = "price", nullable = false)
+    private BigDecimal price;
```

Comment:
these columns have a "name" even though removing it would have the same result. Not sure if it's a standard now, but I would delete these names to reduce boilerplate.